.jenkins/pytorch/win-build.sh

```diff
@@ -118,6 +129,8 @@ set CMAKE_GENERATOR=Ninja
 
 if not "%USE_CUDA%"=="1" (
   if "%REBUILD%"=="" (
+    set CC=sccache clang-cl
+    set CXX=sccache clang-cl
```

Comment:
Does `clang-cl` only work for CPU builds and not CUDA builds? If not we should probably change the `set CC=sccache cl` and `set CXX=sccache cl` lines above instead.

Reply:
@yf225 It will fallback to cl.exe.